lib/internal/constants.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
```

Comment:
Does it make sense to group the constants a little more?  This module looks like it's a place where more constants could be added so maybe it makes sense to create to group them in a `character` or `char` object?

Reply:
In general I agree but I personally think we should do that when it is time to do so instead of adding something before we need it.

Reply:
@kfarnung I agree with @BridgeAR. Besides, I want to refactor similar places in the codebase, so I think I'll do it next time.

Reply:
If we're planning to add more in the future, it makes the scope of any future change smaller and less risky.  I don't have a strong feeling either way, mostly just and observation since the new file is simply `internal/constants.js` and in my experience those sorts of files get out of control before you know it.